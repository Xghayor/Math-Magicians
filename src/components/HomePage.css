.home-container {
    max-width: 800px;
    margin: 7.8% auto;
    padding: 2rem;
    text-align: center;
    background-color: #f8f8f8;
    border-radius: 10px;
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
  }
  
  .home-title {
    font-size: 2rem;
    margin-bottom: 1rem;
    color: #333;
  }
  
  .home-content p {
    font-size: 1.2rem;
    line-height: 1.6;
    color: #555;
  }
  