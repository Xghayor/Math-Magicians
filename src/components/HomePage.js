import React from 'react';
import './HomePage.css'; // Import your custom styles

const HomePage = () => (
  <div className="home-container">
    <h2 className="home-title">Welcome to Math Magicians</h2>
    <div className="home-content">
      <p>
        Explore the world of mathematics with Math Magicians. Whether you are a
        student, educator, or just someone curious about math, you will find
        fascinating topics, interactive tools, and helpful resources here.
      </p>
      <p>
        Explore the world of mathematics with Math Magicians. Whether you are a
        student, educator, or just someone curious about math, you will find
        fascinating topics, interactive tools, and helpful resources here.
      </p>
    </div>
  </div>
);

export default HomePage;
