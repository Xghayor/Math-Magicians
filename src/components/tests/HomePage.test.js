import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../HomePage';

test('Calculator component matches snapshot', () => {
  const { asFragment } = render(<HomePage />);
  expect(asFragment()).toMatchSnapshot();
});
