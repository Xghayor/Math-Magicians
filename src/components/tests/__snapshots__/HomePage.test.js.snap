// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="home-container"
  >
    <h2
      class="home-title"
    >
      Welcome to Math Magicians
    </h2>
    <div
      class="home-content"
    >
      <p>
        Explore the world of mathematics with Math Magicians. Whether you are a student, educator, or just someone curious about math, you will find fascinating topics, interactive tools, and helpful resources here.
      </p>
      <p>
        Explore the world of mathematics with Math Magicians. Whether you are a student, educator, or just someone curious about math, you will find fascinating topics, interactive tools, and helpful resources here.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
